### PR TITLE
fixed inaccuracies in gene queries and updated drug queries

### PR DESCRIPTION
**Pull request by mzheng17**

### CODE REVIEW

File: biolink/pieces-parts/rna-seq-test.rkt [hunk 1]
```diff
@@ -168,232 +219,93 @@ purpose: to find any regulatory relationships where genes on a given list regula
...
-  expression-location
-  )
-#|Examples
-(define cell/tissue-types-mini-test (expression-locations (take uniprots 20)))
-(define cell/tissue-types-full-test (expression-locations uniprots))
+(define 1-hop-mini-test (time (1-hop-query/report (take uniprots 100))))
+(define 1-hop-results (time (1-hop-query/report uniprots)))
 |#
 
+#|
```
Comment (namin) [inline biolink/pieces-parts/rna-seq-test.rkt:292]: Why is this commented out?

**Review by namin — APPROVED** [review]
I am going to merge this, thanks. I find it confusing how your workflow means that REPL commands are springled through. Would be good to have a final showcase of what you need to run in the REPL to get the results.

File: biolink/pieces-parts/rna-seq-test.rkt (hunk 1 above)
Comment (mzheng17) [reply biolink/pieces-parts/rna-seq-test.rkt:292]: I haven't finished testing find-intersections, but I'll send another pull request once I finish that function

**Comment by namin** [comment]
Once you're done with that, can you also have a pull request to consolidate how you run the file. At the minimum, have the full extent of what you run in a comment at the beginning or end, rather than interspersed. Or perhaps even nicer you could create a companion run file, where you do the running? If this does not make sense, we'll discuss with @webyrd next time.

**Comment by mzheng17** [comment]
Yeah, I can include that as well

**Pull request merged**